lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
@brianparry : thanks for your explanation. I just wondering how this line of code works, because of the require caching, normally you should not get a new version of injector. But depends on your testing, this should work. Maybe some of my understanding is not correct.